broker/src/scheduler/active_calls_gc.erl

```diff
@@ -31,22 +31,30 @@ handle_cast(_Msg, State) ->
 
 % TODO: remove duplication with session:notify_status_to_callback_url
 notify_failed_to_callback_url(Call) ->
-  case Call#call_log.callback_url of
+  Project = project:find(Call#call_log.project_id),
+  {StatusUrl, StatusUser, StatusPass, StatusIncludeVars} = project:status_callback(Project),
```

Comment:
The "cached" `callback_url` in the `call_logs` was merely `project.callback_url` and thus missing the other parts (i.e. the basic http auth). I thought it was simpler & better to query the project when needed by the GC (ideally _never_) at the expense of one SQL query, rather than generating the full URL, everytime, for each call.